spec/models/order_spec.rb

```diff
@@ -1,9 +1,29 @@
 require "rails_helper"
 
 RSpec.describe Order, type: :model do
-  let(:order) { build(:order) }
+  subject { build(:order) }
 
   it "has a valid factory" do
-    expect(order).to be_valid
+    expect(subject).to be_valid
```

Comment:
**Nitpick - Code Quality**
A better spec would be `it { is_expected.to be_valid }`. Since we are asserting `subject`.